src/javascript/therapyCode.js

```diff
@@ -1,358 +1,86 @@
-			teacher: {min: 3, max: 100},
-			friend: {min: 100, max: 100}
-		},
-		allowHypno: function() {
-			return false;
-		},
-		check: function() {
-			return true;
-		},
-		finishSession: function() {
-			State.active.variables.player.perversion.friend=5;
-		},
-		finishHypno: function() {}
-	},
-	{	//Talking about Friend - massage oils
-		text: "You tell $therapist how much the massages with $friend are relaxing you and helping you sleep at night. She beams at you and says, @@.therapist;\"Oh that's wonderful $player.name! It's so nice to know I've been able to help you. And I think it's so nice that you've been able to return $friendG.his_her generosity.\"@@\n\nShe's thoughtful for a moment.\n\n@@.therapist;\"Massaging someone is quite an intimate act. How are you coping with that aspect?\"@@ You confess that your relationship has become somewhat intimate, what with rubbing the oils in, seeing $friend naked from behind, and even seeing $friendG.him_her wearing your old butt-plug! @@.hero;Oops...@@\n\n$therapist simply nods.\n\n@@.therapist;\"Intimacy is very important to someone at your age, and I think exploring ways to increase that intimacy will lead you to greater happiness,\"@@ she says, smiling at you. @@.therapist;\"You and $friend enjoy playing games together, and I recommend making those games a bit more intimate as well. For example, you both wear butt-plugs from time to time it seems, so maybe think of a game you could play involving them?\"@@",
-		hasPassage: false,
-		chance: 10,
-		priority: 5,
-		forceHypno: false,
-		kinkHypno: true,
-		allowTalks: true,
-		perversion: {
-			guardian: {min: 0, max: 100},
-			therapist: {min: 2, max: 100},
-			teacher: {min: 3, max: 100},
-			friend: {min: 100, max: 100}
-		},
-		allowHypno: function() {
-			return false;
-		},
-		check: function() {
-			return true;
+	{	//start of routine
+		text: "Curious about your newfound interest in men, you ask if this research study could cause something like that?\n\n@@.therapist;\"Hmm, it seems you may have a very low resistance to this kind of material, which is interesting. Don't worry – if you experience any side effects, they should be temporary. The first session was a probe, so to speak, meant to gauge your reaction so I can calibrate the content to suit you. If you choose to continue in future sessions, you shouldn't have those side effects, and we'll instead focus on helping you feel more relaxed, and making it easier to deal with the stress in your life.\"@@\n\nWell, that last session definitely felt weird, but you have already seen so much porn in your life do not think participating in $therapist's research will affect you in the long run. It would surely make dealing with paying the tutition easier.\n\nNow that the immediate financial issues are sorted out, you can talk to $therapist about your personal experiences. I have allocated an additional time-slot so you can participate in my study after your personal councelling.",
```

Comment:
```suggestion
		text: "Curious about your seemingly newfound interest in men, you ask if this research study could cause something like that?\n\n@@.therapist;\"Hmm, it seems this kind of material may be awakening some latent desires, which is interesting. Don't worry – if you experience any side effects, they should be temporary. The first session was a probe, so to speak, meant to gauge your reaction so I can calibrate the content to suit you. If you choose to continue in future sessions, you shouldn't have those side effects, and we'll instead focus on helping you feel more relaxed, and making it easier to deal with the stress in your life.\"@@\n\nWell, that last session definitely felt weird, but you have already seen so much porn in your life that you do not think participating in $therapist's research will affect you in the long run. It would surely make dealing with paying your tutition easier.\n\nNow that the immediate financial issues are sorted out, you can talk to $therapist about your personal experiences. I have allocated an additional time-slot so you can participate in my study after your personal councelling.",
```